pkg/storage/hack/main.go

```diff
@@ -84,7 +85,9 @@ func getStore() (lstore.Store, error) {
 }
 
 func fillStore() error {
-	store, err := getStore()
+	cm := storage.NewClientMetrics()
```

Comment:
Two nits:
1. Can we inject `clientMetrics` into `fillStore()`?. I see it's used only in main. Can we create clientMetrics in main and inject into both getStore and fillStore()? wdyt?
2. Then we can move the `defer Unregister()` directly into `main()`?